vars/Fedora-25.yml

```diff
@@ -3,4 +3,7 @@
 # Put internal variables here with Fedora 25 specific values.
```

Comment:
Fedora 25 is very obsolete, and any future Fedora version will be soon. Please use just Fedora.